packages/lit-html/src/directives/style-map.ts

```diff
@@ -66,10 +66,6 @@ class StyleMapDirective extends Directive {
 
     if (this._previousStyleProperties === undefined) {
```

Comment:
This entire `if` can go away and instead we start with L28 initializing `_previousStyleProperties: Set<string> = new Set();`